runtime/src/accounts_db.rs

```diff
@@ -2253,7 +2388,7 @@ impl AccountsDb {
     }
 
     pub fn load_slow(&self, ancestors: &Ancestors, pubkey: &Pubkey) -> Option<(Account, Slot)> {
-        self.load(ancestors, pubkey)
+        self.load(ancestors, pubkey, false)
```

Comment:
Hmm, this is called from `get_sysvar_account()` which could be called via the code path of bank-hash calculation like `update_sysvar_account`. So, I think we need this flag switch one or two higher level... ;)

Reply:
This is a good catch, updated here: https://github.com/solana-labs/solana/pull/15454/commits/9d8a76a5306a6fe53e1f2cdbc2fb3152a6414510